### ISSUE TITLE
Tutorial zip broken

### ISSUE THREAD

**Issue by Rinze-CodeGorilla** [angular/angular#57075]
### Describe the problem that you experienced

The Angular tutorial requires downloading the project files to get started, it provides a link to a .zip file. This .zip file is broken: the files in the base directory are instead marked as being in the root directory, which means they don't extract by default, and in some tools aren't even visible. Not having the files in the base directory means I can't run npm install or ng serve because the package.json and angular configuration files are missing.

The zip files are broken in all lessons, not just the first one.

### Enter the URL of the topic with the problem

https://angular.dev/tutorials/first-app/01-hello-world

### Describe what you were looking for in the documentation

I want to learn Angular

### Describe the actions that led you to experience the problem

Open tutorial and follow step 1 to download the files. 

> Start by clicking the "Download" icon in the top right pan of the code editor. This will download a .zip file containing the source code for this tutorial.

### Describe what you want to experience that would fix the problem

A zip that has it's files in the right location so they extract to the right location and I can actually learn Angular

### Add a screenshot if that helps illustrate the problem

Windows:
![Screenshot 2024-07-22 112346](https://github.com/user-attachments/assets/b8e4345d-1349-46da-960d-9a9f200c80e0)
Notice the extra \ to view the files from the root (should be base) directory
![Screenshot 2024-07-22 112400](https://github.com/user-attachments/assets/32249f19-28b7-4e0f-bfd8-2a59c8a52b54)
Linux
![Screenshot 2024-07-22 113046](https://github.com/user-attachments/assets/37ca6ad1-b1fa-45ee-9f38-40f6681494a3)

**Comment by 9kubczas4** [comment]
Hi @JoostK,

I can take a look on this issue.

**Comment by 9kubczas4** [comment]
Hi,

I tried to reproduce this issue, however structure of the files looks correct for me.
![image](https://github.com/user-attachments/assets/f9caba88-6b37-492b-ad65-2f7bfeb9f6ec)
I was able to install dependencies and run app via `ng serve` successfully.

I generated the zip using:
- macOS 14.5
- NixOS - Kernel 6.10

@Rinze-CodeGorilla could you tell me which exact OS version do you use and what tool to unzip is your choice?

**Comment by 9kubczas4** [comment]
Yes, I assume that probably the issue is platform-dependent. I'll continue the investigation of it.

**Comment by kuselj** [comment]
@mhevery @9kubczas4, and @Everyone This is very concerning to see that things have been broken since Jul 2024 - this is a person's first impression of Angular and for those contemplating React, Vue etc. this will rather push them in those directions (also mentioned in the online forums etc.) 

I recall trying Angular back in Oct as well ran into these kind of issues and thought it was just maybe growing pains of recent release.

Or is Angular being replaced by something else and this is broken windows scenario?

**Comment by JeanMeche** [comment]
The issue is indeed platform/software dependent. I wasn't able to reproduce it but I might have an idea in #59726. 

Could someone check if the issue is fixed on https://ng-dev-previews-fw--pr-angular-angular-59726-adev-prev-lv9mcx4y.web.app/playground ?

**Comment by kuselj** [comment]
Thanks @JeanMeche 

When you say platform/software issue you, you are referring to the platform/software used for doing the build/packaging of the source code that is then published for the tutorial? (Since it can't be a client platform/software issue - the source code was missing elements and we don't specify/choose/select OS for download of the source etc.)

As a user I can say & confirm the playground Hello World download seems to be working (on my Windows 11) which is good :)

**Clearly the Angular Team needs to validate & QA the fix as well as the Tutorials from end to end and update where applicable on the Angular side**

But thank you guys for getting the ball rolling on this - appreciate it!

**Comment by JeanMeche** [comment]
The issue is likely due how the unzip tool behaves. 

When inspecting the archive all the files were there, my fix just removes a leading slash on some paths that are interpreted differently as it seems.
